db/migrate/20160512012637_add_thumbnail_to_serials.rb

```diff
@@ -0,0 +1,5 @@
+class AddThumbnailToSerials < ActiveRecord::Migration
+  def change
+    add_column :serials, :thumbnail, :string, null: false
```

Comment:
Additionally, we **shouldn't** share post's thumbnail with serial.
When post is destroyed, the file is also destroyed from storage by CarrierWave.
Then serial's thumbnail has gone unexpectedly.